src/ui/TimeSpanPicker.tsx

```diff
@@ -39,6 +39,7 @@ type Props = {
   onChangeStartTime: (newStartTime: string) => void;
   onChangeEndTime: (newEndTime: string) => void;
   disabled?: boolean;
+  minWidth?: string;
 } & InlineProps;
```

Comment:
```suggestion
  disabled?: boolean;
} & InlineProps;
```

Isn't it already included in `InlineProps`

Reply:
True 👍